src/ports/postgres/modules/deep_learning/madlib_keras_wrapper.py_in

```diff
@@ -160,7 +172,7 @@ def parse_and_validate_compile_params(str_of_args):
         opt_args:               Arguments for the optimizer
         compile_dict:           Dictionary of arguments for keras.compile
     """
-
+    _assert(str_of_args, "Compile parameters cannot be empty or NULL.")
```

Comment:
True, the value here will not be NULL, but the value passed by the user can be either empty or NULL. Since we convert a `None` to empty string in the other code you pointed out, we won't know the original value exactly here. So made the error message generic.